src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssKeysAndValues = sourceString
+    .split(';')
+    .filter((item) => item.trim().length)
+    .reduce((prev, item) => {
+      const index = item.indexOf(':');
+
+      return {
+        ...prev,
+        [item.slice(0, index).trim()]: item.slice(index + 1).trim(),
```

Comment:
[CODE KNOWLEDGE] - Don't use iteration methods to modify an already existing array/object. The spread operator in the return statement of the reduce callback is not optimal for performance because it creates a shallow copy of the object in each iteration. Consider using the object reference directly to assign new properties.